python/triqs/lattice/utils.py

```diff
@@ -246,3 +246,161 @@ def TB_from_pythTB(ptb):
                     orbital_names=[str(i) for i in range(ptb.get_num_orbitals())])
 
     return TBL
+
+# importing necessary dependencies
+from itertools import product as itp
+from pythtb import *
+# from triqs.lattice.tight_binding import TBLattice # TRIQS needs to be installed for this dependency to work
+import sympy as sp
+from sympy import *
+import warnings
+
+def sympyfy(tb_lat_obj, analytical = True):
+    r"""
+    returns the analytical form of the momentum space hamiltonian of the tight-binding model,  
+    from a tight-binding lattice object, by utilizing Fourier series
+    
+    Parameters
+    ----------
+    tb_lat_obj: triqs TBLattice object
+        triqs tight binding object
+    analytical: boolean, default = True
+        a boolean which will cause the function will return an analytical Hamiltonian, when true, and 
+        an numerical Hamiltonian otherwise
+    
+    Returns
+    -------
+    Hk_numerical: NumPy Array
```

Comment:
This looks as if the function gives back to values, but it only gives back one value. Just list in Return one object and explain in the text that the object given back depends on the `analytical` flag.

Reply:
Fixed.